src/components/Overview/Overview.react.js

```diff
@@ -147,12 +146,12 @@ class Overview extends Component{
               &nbsp;and <b>iOS devices</b>.
                    Download the App to have access to SUSI on the go.</p>
                    <div className="description__text store_icons">
-                     <Link className="playstore"
-                       target="_blank" to="https://play.google.com/store/apps/details?id=org.fossasia.susi.ai"> Get it on Google Play
-                     </Link>
-                     <Link className="appstore"
-                       target="_blank" to="https://github.com/fossasia/susi_iOS"> Download on the App Store
-                     </Link>
+                     <a className="playstore"
+                       target="_blank" rel="noopener noreferrer" href='https://play.google.com/store/apps/details?id=org.fossasia.susi.ai'> Get it on Google Play
```

Comment:
Please correct the link to https://play.google.com/store/apps/details?id=ai.susi.

Reply:
Done, please check